website/docs/language/values/variables.mdx

```diff
@@ -489,7 +489,7 @@ you can use the [`-compact-warnings`](/terraform/cli/commands/plan#compact-warni
 option to simplify your output.
 
 If you provide values for undeclared variables on the [command line](#variables-on-the-command-line),
-Terraform will error. To avoid this error, either declare a variable block for the value, or remove
+Terraform will throw an error. To avoid this error, either declare a variable block for the value, or remove
```

Comment:
Consider changing "throw" to "return"? 

My thinking is that this is the command line process exiting with a non-zero status, versus an exception that can be thrown and caught. In this same document we previously say "return an error," so that feels more consistent with the existing document. Thanks!